packages/sdk/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@waku/create",
-  "version": "0.0.14",
-  "description": "Easily create a Waku node",
+  "name": "@waku/sdk",
+  "version": "0.0.15",
```

Comment:
Hum, so the idea is to manual release after this PR is merged? Sounds good, let's be sure you then tell release please to start from this version.

Reply:
This one requires us to add `@waku/sdk` to [the list of packages being tracked](https://github.com/waku-org/js-waku/blob/259b39f207062b312c2900aa1c5297bb59fc5f9c/release-please-config.json#L8) and [to manually set next version](https://github.com/waku-org/js-waku/pull/1367/files#diff-c55d4dcb68c348b2c06d263819702fbce72eeeb35b662956d02049a5a18fcf2b).